{{cookiecutter.project_shortname}}/site/setup.cfg

```diff
@@ -0,0 +1,51 @@
+
+[metadata]
+name = {{cookiecutter.project_shortname}}
+version = 1.0.0
+description = {{cookiecutter.description}}
+long_description = file: README.rst, CHANGES.rst
+license = MIT
+author = {{cookiecutter.author_name}}
+author_email = {{cookiecutter.author_email}}
+platforms = any
+url = {{cookiecutter.github_repo}}
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >={{cookiecutter.python_version.split()[0]}}
+zip_safe = False
+
+[options.extras_require]
+tests =
+    pytest-invenio>=2.1.0,<3.0.0
+
+[options.entry_points]
+invenio_base.blueprints =
+    {{cookiecutter.project_package}}_views = {{cookiecutter.project_package}}.views:create_blueprint
+invenio_assets.webpack =
+    {{cookiecutter.project_package}}_theme = {{cookiecutter.project_package}}.webpack:theme
+invenio_config.module =
```

Comment:
Isn't this gonna cause confusion? Having both invenio.cfg and this config in the template folder?